src-web/tableDefinitions/policyDetailsOverviewDef.js

```diff
@@ -107,5 +107,13 @@ export default {
         }
       ]
     },
+    {
+      cells: [
```

Comment:
Can you remove this? I asked @willkutler to remove we already have the link in main table as first class citizen.

Reply:
Do you mean this or the one in the table action in grcPoliciesViewDef.js, I can remove that but remove this one will disable drill down link

Reply:
Sorry, I meant the one in main table.